### PR TITLE
Allow display of compositor statistics from Dart

### PR DESCRIPTION
**Pull request by chinmaygarde**

### CODE REVIEW

File: sky/engine/core/compositing/SceneBuilder.idl [hunk 1]
```diff
@@ -14,6 +14,7 @@
   void pop();
 
   void addPicture(Offset offset, Picture picture, Rect bounds);
+  void addStatistics(unsigned long enabledOptions, Rect bounds);
```
Comment (abarth) [inline sky/engine/core/compositing/SceneBuilder.idl]: How does the caller know what bounds to use?  I guess you pass the bounds of the whole screen and hope the stats fit?

**Comment by abarth** [comment]
LGTM

File: sky/compositor/compositor_options.h [hunk 2]
```diff
@@ -21,6 +21,8 @@ class CompositorOptions {
   };
 
   CompositorOptions();
+  CompositorOptions(uint64_t mask);
```
Comment (jamesr) [inline sky/compositor/compositor_options.h]: our typical C++ style would ask for `explicit` here

File: sky/engine/core/compositing/SceneBuilder.idl (hunk 1 above)
Comment (chinmaygarde) [inline sky/engine/core/compositing/SceneBuilder.idl]: For the text, pretty much :/ But for the graphs, they will resize to fill the available bounds. Another PR incoming for that.

**Comment by chinmaygarde** [comment]
Made the ctor `explicit`

**Pull request merged**